docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,232 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker. 
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability. 
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point of deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service (you can choose one of them)
+    * DynamoDB
+    * RDS (MySQL/PostgreSQL)
+* 1 Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards. 
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production. This section will help you to configure a secure network for Scalar DL deployments.
+
+### Requirements
+ 
+* You must have a VPC, with NAT gateways on private subnets. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+* You must have at least 2 subnets in different AZs for the Kubernetes cluster. This is mandatory to create an EKS cluster.
+* You must have at least one private subnet for the managed database other than Dynamo DB.
```

Comment:
`the managed database ` is RDS in this case?
Can you make it more specific if so?
Or should it be `managed databases`?

Reply:
> Please answer the question.

This is a general guide to set up a database in AWS.

> the managed database is RDS in this case?

Yes. mentioned that in the sentence.

> Or should it be managed databases?

Yes. Because In this guide we use managed databases.

Reply:
Are you answering to what questions?

Reply:
> the managed database is RDS in this case? 
Or should it be managed databases?

Yes. It is RDS I have mentioned that in the sentence also changed to `managed databases`.